src/main/java/org/mule/module/json/internal/cleanup/JsonModuleResourceReleaser.java

```diff
@@ -84,28 +84,17 @@ public synchronized void releaseExecutors() {
     } catch (NoSuchFieldException | IllegalAccessException | InterruptedException ex) {
       LOGGER.error("Caught exception while stopping the Executor Service references of the MessageBundles class: {}",
                    ex.getMessage(), ex);
-    } finally {
-      if (bundleField != null) {
-        bundleField.setAccessible(isAccessible);
-      }
     }
 
     bundleField = null;
-    isAccessible = false;
 
     try {
       bundleField = ProcessingMessage.class.getDeclaredField("BUNDLE");
-      isAccessible = bundleField.isAccessible();
-      bundleField.setAccessible(true);
       MessageBundle messageBundle = (MessageBundle) bundleField.get(null);
       cleanMessageBundle(messageBundle);
     } catch (NoSuchFieldException | IllegalAccessException | InterruptedException ex) {
```

Comment:
I recognize that the handling of InterruptedException differs from the other exceptions in this catch block and that **this observation is not part of your changes**. However, I believe it's important to address, as currently, the InterruptedException is being ignored. Ignoring this exception can lead to potential issues, such as making the application unresponsive or complicating the process of shutting down gracefully. It might be more effective to handle this exception in a way that respects thread interruption, perhaps by restoring the interrupt status with Thread.currentThread().interrupt();. This adjustment would allow other parts of the application to properly respond to the interruption. I understand this wasn't part of your initial modifications, but what are your thoughts on making this change? 
**_(The same observation applies to lines 71 and 84)._**